src/session/session_api.c

```diff
@@ -1668,16 +1667,6 @@ __wt_session_range_truncate(
     trunc_info->orig_stop_key = orig_stop_key;
     trunc_info->uri = actual_uri;
 
-    /*
-     * Don't use bounded cursors for FLCS as it isn't supported, additionally skip using them for
-     * complex types such as column groups and indexes. We can't check support for those complex
-     * types at this abstraction level.
-     */
-    if (CUR2BT(start) == NULL || CUR2BT(start)->type == BTREE_COL_FIX)
-        supports_bounds = false;
-    if (stop != NULL && (CUR2BT(stop) == NULL || CUR2BT(stop)->type == BTREE_COL_FIX))
-        supports_bounds = false;
-
     /*
      * Truncate does not require keys actually exist so that applications can discard parts of the
```

Comment:
Please update this comment so that it doesn't mention bounded cursors, but keep the description of prepare conflicts.

Reply:
Thanks! Fixed in [144c765](https://github.com/wiredtiger/wiredtiger/pull/10825/commits/144c7656cf3422d4b8747c078104bc9f0dd58c32)